src/deadline/job_attachments/download.py

```diff
@@ -256,6 +273,62 @@ def _get_tasks_manifests_keys_from_s3(
     return manifests_keys
 
 
+def update_manifest_output_paths_for_session_actions(
```

Comment:
Let's keep this function prefixed with `_` until we've used it as an internal function of the library for a while. That way we can make changes, and then remove the `_` once we're confident in the interface.